app/services/open_library_service.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class OpenLibraryService
+  include HTTParty
+  HTTPARTY_BASE_URL = 'https://openlibrary.org' # tira quilombos con rubocop
+  base_uri HTTPARTY_BASE_URL
+
+  def book_hash(isbn)
```

Comment:
This name for the method if kind non descriptive.

You should use something like `get_books`, describing the action to be done in the request.